app/controllers/omninav_controller.rb

```diff
@@ -5,7 +5,31 @@ def demo
     render html: omninav_html, layout: 'omninav'
   end
 
+  # GET /omni_nav/omni_nav_v2(/:context)
   def demo_v2
-    render template: "omni_nav/omni_nav_v2", layout: 'omninav'
+    # params[:context] can be used to simulate different contexts (e.g. /omninav_v2/law)
+    @theme_class = params[:context]
+
+    # Apparently this is a thing in Cascade. Kill me. See for instance the law home pages
+    # where DOM tree goes:
+    # body > div#theme.law > ... > div.cu-off-canvas-header > a.sc-logo.fowler
+    theme_logo_map = {
+      # theme class => logo class
+      'business' => 'asbe',
+      'law' => 'fowler'
+    }
+    @logo_class = theme_logo_map.fetch(@theme_class, 'default-logo-cu')
+
+    # I believe the identification of the following variables will have to be added
+    # to Cascade (in the velocity format?) for OmniNav v2.
+    theme_brand_map = {
+      # theme class => brand_text
+      'law' => 'Fowler School of Law'
+    }
+    @brand_text = theme_brand_map.fetch(@theme_class, '')
+
+    @brand_class = params[:context].nil? ? 'unbranded' : 'branded'
```

Comment:
This (minimally) recreates the context-switching for off-canvas nav that we have in Cascade. Note the last section, which may requires some tweaks to velocity switch that is used in current version of OmniNav.